releasenotes/notes/1.3/dag-oxide-60b3d7219cb21703.yaml

```diff
@@ -28,8 +28,8 @@ upgrade_transpiler:
 
     however this type of mutation was always unsound as it could break the DAG's internal caching
     and cause corruption of the data structure. Instead you should use the API provided by
-    :class:`.DAGCircuit` for mutation such as :meth:`.DAGCircuit.substitute_node`,
-    :meth:`.DAGCircuit.substitute_node_with_dag`, or :meth:`.DAGCircuit.contract_node`. For example
```

Comment:
What is the reason for removing this from the release notes?

Reply:
`DAGCircuit.contract_node` does not exist 🙂 That was probably a confusion coming from petgraph's `StableGraph::contract_node`, but this is not exposed on the `DAGCircuit`.